db/contacts.json

```diff
@@ -0,0 +1,93 @@
+{
+	"contacts": [{
```

Comment:
This should be an object of objects with keys for each contact. And, hey, while you are editing this, go ahead and add uid to the contacts 😄